portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro

```diff
@@ -721,6 +729,18 @@ definition {
 		}
 	}
 
+	macro viewFieldReferenceFromMultipleSelection {
+		Click(
+			key_fieldName = "Multiple Selection",
+			locator1 = "Form#FIELD_BODY");
+
+		Form.gotoAdvancedTab();
+
+		AssertTextEquals(
+			locator1 = "Form#FIELD_REFERENCE",
+			value1 = "${key_fieldReference}");
+	}
+
```

Comment:
Hi @ewertonluna , I saw that there is a similar macro called `viewFieldReferenceFromTextField` [here](https://github.com/liferay/liferay-portal/blame/master/portal-web/test/functional/com/liferay/portalweb/macros/Form.macro#L690-L701) but placed in another file. 

Could you try to reuse the `viewFieldReferenceFromTextField` by renaming it to `viewFieldReference` and receiving the `fieldName` as a parameter? Maybe you could also move this macro to the `FormFields.macro` file which I think is the best place.

Thanks!

Reply:
Hi @carolmariaabb! I just made the necessary changes you asked. Could you please check again and let me know if there are any remaining issues? Thank you!